src/components/TimeSeriesCard/TimeSeriesCard.jsx

```diff
@@ -68,6 +73,17 @@ const TimeSeriesCardPropTypes = {
     ),
     /** optional units to put in the legend */
     unit: PropTypes.string,
+    /** Optionally addes a zoom bar to the chart */
+    zoomBar: PropTypes.shape({
+      /** Determines which axis to put the zoomBar */
+      axes: PropTypes.string,
```

Comment:
I'm not familiar with the prop model that much. Is there a specific set of axes? What format should the string be in? Wondering if we should make it an enum for clarity
```
axes: PropTypes.oneOf(['x-axis', 'y-axis']),
```
or
```
axes: PropTypes.oneOf(['top', 'bottom', 'left']),
```

Reply:
I think we also would need to update this in `CardPropTypes`

Reply:
Oh good call. I'll update it to be `axes: PropTypes.oneOf(['top', 'bottom', 'left']),` , where I could actually enforce the `'top'` until it the other 3 are supported